test-project-scaffolding/Doist.Integrations.TestProjectBootstrap/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Octokit;
+using System;
+using System.IO;
+using System.Management.Automation;
+using System.Threading.Tasks;
+
+namespace Doist.Integrations.TestProjectBootstrap
+{
+    class Program
+    {
+        const string FeatureBranchName = "features/labels-sync";
+        const string PRDescriptionFileName = "PR_DESCRIPTION.md";
+
+        static async Task Main(string[] args)
+        {
+            // var directory = @"C:\Users\krato\src\integrations-test-project";
+
+            Console.WriteLine("Welcome to the Doist Integrations' test project bootstrapping tool.");
+            Console.WriteLine();
+
+            Console.WriteLine("Path to your local Git repository:");
+            Console.WriteLine();
+            var localRepositoryPath = Console.ReadLine();
+
+            Console.WriteLine();
+            Console.WriteLine("Your GitHub token:");
+            var gitHubToken = Console.ReadLine();
+
+            Console.WriteLine();
+
+            while (true)
+            {
+                Console.WriteLine("Applicant name, written as applicant-name. If you're done, press Enter");
```

Comment:
```suggestion
                Console.WriteLine("Applicant name, written as firstname-lastname. If you're done, press Enter");
```

Is that what you mean?